guides/common/modules/proc_importing-kickstart-repositories.adoc

```diff
@@ -114,9 +108,9 @@ ifeval::["{rhel-version}" == "8"]
 * Change `repository = BaseOS` to `repository = .`
 . Delete the `[media]` and `[variant-AppStream]` sections.
 . Save and close the file.
-. Verify that the `/var/www/html/pub/sat-import/content/dist/rhel8/8.1/x86_64/baseos/kickstart/treeinfo` file has the following format:
+. Verify that the `/var/www/html/pub/sat-import/content/dist/rhel{el-major-version}/{el-major-version}.{el-minor-version}/x86_64/baseos/kickstart/treeinfo` file has the following format:
 +
-[options="nowrap" subs="+quotes"]
+[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 [checksums]
 images/efiboot.img = sha256:9ad9beee4c906cd05d227a1be7a499c8d2f20b3891c79831325844c845262bb6
```

Comment:
This is technically wrong, but I have no way of getting those checksums for RHEL9. I am not sure if anyone cares, though.

Reply:
I doubt that anyone cares either, but if you want here are the checksums for RHEL9:
```
[checksums]
images/efiboot.img = sha256:c01c18acc6778d6e66c8d0872bac59bfd7219ccf3cfa70a5c605c0fb37f33a83
images/install.img = sha256:ddd08e5a5d92edee150f91ff4f12f39253eae72ff496465cf1b2766fe4a4df49
images/pxeboot/initrd.img = sha256:a09a8ec89d485d71ed1bdad83584d6d816e67448221172d9aad97886cd70adca
images/pxeboot/vmlinuz = sha256:6e523d7c3266e26c695923ab12b2873b16b0c61fb2e48ade608ad8998821584b
```

Reply:
Thanks for looking those up. Applied.